frontend/src/pages/notebookController/StartServerModal.tsx

```diff
@@ -1,11 +1,25 @@
 import * as React from 'react';
-import { Alert, AlertVariant, Button, Modal, ModalVariant, Spinner } from '@patternfly/react-core';
-import { Notebook } from '../../types';
-import { checkNotebookRunning } from '../../utilities/notebookControllerUtils';
+import {
+  Alert,
+  AlertVariant,
+  Button,
+  ExpandableSection,
+  List,
+  ListItem,
+  Modal,
+  ModalVariant,
+  Progress,
+  ProgressVariant,
+} from '@patternfly/react-core';
+import { checkNotebookRunning, getNotebookStatus } from '../../utilities/notebookControllerUtils';
+import { useWatchNotebookEvents } from '../../utilities/useWatchNotebookEvents';
+import NotebookControllerContext from './NotebookControllerContext';
+import { EventStatus, Notebook } from '../../types';
 
 import './NotebookController.scss';
 
 type StartServerModalProps = {
+  projectName: string;
```

Comment:
We'll want to pull this from a [common hook](https://github.com/opendatahub-io/odh-dashboard/pull/333/files#diff-7b4aa47a37ae24d16cc129957c6bf3cc76e4fba423794e43e28760aa431f6dd4) after my PR merges... but you can leave it for now. Something to consider moving forward.

Reply:
Will update it after rebase.